x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -532,7 +532,6 @@ export const waitForAlertsToPopulate = async (alertCountThreshold = 1) => {
   cy.waitUntil(
     () => {
       refreshPage();
-      cy.get(LOADING_INDICATOR).should('exist');
```

Comment:
@yctercero is there any reason why this line has been deleted? This was added to fix some tests and remove flakiness